XamlControlsGallery/ControlPages/RevealFocusPage.xaml

```diff
@@ -108,66 +108,64 @@
 
             <StackPanel>
                 <StackPanel Orientation="Horizontal">
-                    <Button x:Name="exampleButton"
+                    <Button x:Name="exampleButton"  AutomationProperties.Name="Square button 4"
```

Comment:
Nit: since the rest of the properties are on new lines, please also move this to a new line for readability.

Reply:
Ctrl+K, Ctrl+D in Visual Studio will automatically format the markup.